Gemfile.lock

```diff
@@ -296,8 +296,24 @@ GEM
     rb-inotify (0.10.0)
       ffi (~> 1.0)
     redis (4.1.3)
+    redis-actionpack (5.2.0)
```

Comment:
Reading the docs it does kind of imply we only need `redis-actionpack` but I don't think it is an issue that should block this - just interested to hear your take.

Reply:
I wondered this but when looking at redis-actionpack they link back to redis-rails if it's being used in a Rails context:

> If you are using redis-store with Rails, head on over to the redis-rails README to learn how to integrate this gem into your Rails application.

https://github.com/redis-store/redis-actionpack#usage

Now it works, I will test if only using actionpack works and report back as I agree less dependencies the better!

Reply:
All works! Done 👍